go.mod

```diff
@@ -1,6 +1,4 @@
-go 1.21
-
-toolchain go1.21.0
```

Comment:
how come this was removed?

Reply:
My Go's `go mod tidy` did that, let me manually revert. Sorry about that.

Reply:
oh all good, this forces the usage of go 1.12 from my knowledge. what settings do you have on your go mod?